pkg/agent/cluster.go

```diff
@@ -70,9 +71,22 @@ func NewCluster(ctx context.Context, assetDir, rendezvousIP, kubeconfigPath, ssh
 	czero := &Cluster{}
 	capi := &clientSet{}
 
-	authToken, err := FindAuthTokenFromAssetStore(assetDir)
-	if err != nil {
-		logrus.Fatal(err)
+	var authToken string
```

Comment:
As an additional safety measure, wouldn't be better to enable this check iff [AUTH_TYPE is not none](https://github.com/openshift/installer/pull/8108/files)?

Reply:
Once enabled, we are not going to change it none authenticator , isn't it?

Reply:
Not a blocking point for this point, but I think it would be useful to have (internally) a single knob to enable the auth feature (for example, for developing/debugging). Right now we have a branch of code where the feature could be activated (via the env var in assisted-service), and another where it's always enabled (here)

Reply:
I agree. It would be very useful to have a way to turn off the authentication code paths. It would have helped with not breaking the day-2 monitor command.